WcaOnRails/app/views/persons/_personal_records.html.erb

```diff
@@ -0,0 +1,40 @@
+<div class="personal-records">
+  <h3 class="text-center"><%= t 'persons.show.personal_records' %></h3>
+  <div class="table-responsive">
+    <table class="table table-striped">
+      <thead>
+        <tr>
+          <th class="event"><%= t 'competitions.results_table.event' %></th>
+          <th class="country-rank">NR</th>
+          <th class="continent-rank">CR</th>
+          <th class="world-rank">WR</th>
+          <th class="single"><%= t 'common.single' %></th>
+          <th class="average"><%= t 'common.average' %></th>
+          <th class="world-rank">WR</th>
+          <th class="continent-rank">CR</th>
+          <th class="country-rank">NR</th>
+        </tr>
+      </thead>
+      <tbody>
+        <% @events_competed_in.each do |event| %>
+          <% rank_single = @ranks_single.find { |rank| rank.event_id == event.id } %>
+          <% rank_average = @ranks_average.find { |rank| rank.event_id == event.id } %>
+          <tr>
+            <td class="event">
+              <%= cubing_icon event.id %>
+              <%= t "events.#{event.id}" %>
+            </td>
+            <%= rank_td rank_single, "country" %>
```

Comment:
I believe the current PHP code links to the country single rankings for this event here.